.github/workflows/joshua.yml

```diff
@@ -15,12 +15,13 @@ jobs:
       
       - name: Install fdb and dependencies
         run: |
+          sudo mkdir -p /var/lib/foundationdb/data
           wget https://www.foundationdb.org/downloads/6.3.12/ubuntu/installers/foundationdb-clients_6.3.12-1_amd64.deb --no-check-certificate
           wget https://www.foundationdb.org/downloads/6.3.12/ubuntu/installers/foundationdb-server_6.3.12-1_amd64.deb --no-check-certificate
           echo "05b11ac59cb44012e863113fa552c6cf53fb04cfbdb8e72a4a62770cbd2ddd81  foundationdb-clients_6.3.12-1_amd64.deb" >> checks.txt
           echo "15472291c463c617f4f4f2c5e2fcb52ecb08757e481ee7f35e1b352999a7ea99  foundationdb-server_6.3.12-1_amd64.deb" >> checks.txt
           sha256sum -c checks.txt && sudo dpkg -i *.deb
-          sudo pip3 install foundationdb lxml pytest
+          sudo pip3 install foundationdb lxml pytest python-dateutil
```

Comment:
add `subprocess32` and maybe for good measure `psutil` and consider pinning versions for consistency.

Reply:
Done